src/main/java/duke/ui/UI.java

```diff
@@ -0,0 +1,177 @@
+package duke.ui;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+import duke.dukeexception.DukeTaskNonExistException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.deadline.Deadline;
+import duke.task.eventtask.EventTask;
+import duke.task.todo.ToDo;
+import duke.tasklist.TaskList;
+
+/**
+ * Class that deals with interactions with the user.
+ */
+public class UI {
+    /**
+     * Empty line to be printed after each command
+     */
+    private static final String LINE = "************************************\n";
+    private final TaskList taskList;
+    private final Storage storage;
+
+    /**
+     * Constructor for the UI object
+     *
+     * @param taskList   Tasklist
+     * @param storage Storage
+     */
+    public UI(TaskList taskList, Storage storage) {
+        this.taskList = taskList;
+        this.storage = storage;
+    }
+
+    /**
+     * This method shows the welcome message for the chatbot. It is what users see when they launch the chatbot.
+     */
+    public String welcome() {
+
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        StringBuilder sb = new StringBuilder(LINE);
+        sb.append("Hello from \n");
+        sb.append(logo);
+        sb.append("What can I do for you pal? :D\n");
+        sb.append(LINE);
+        return sb.toString();
+    }
+
+
+    /**
+     * Method that prints the goodbye statement
+     */
+    public String replyBye() {
```

Comment:
Perhaps the naming of the method could be improved? 
The method name suggests to me that a "bye" response is printed and that it is a void return method instead of a String return method.
I have noticed the same issue in several places too.